JAICore/jaicore-planning/src/test/java/jaicore/planning/hierarchical/testproblems/nesteddichotomies/CEOCSTNNestedDichotomyProblemSet.java

```diff
@@ -0,0 +1,61 @@
+package jaicore.planning.hierarchical.testproblems.nesteddichotomies;
+
+import jaicore.basic.algorithm.AAlgorithmTestProblemSet;
+import jaicore.planning.hierarchical.problems.ceocstn.CEOCSTNPlanningProblem;
+
+public class CEOCSTNNestedDichotomyProblemSet extends AAlgorithmTestProblemSet<CEOCSTNPlanningProblem>  {
+
+	public CEOCSTNNestedDichotomyProblemSet() {
+		super("Nested Dichotomies");
+	}
+
+	//	private void checkNumberOfSolutionsForProblemSize(final int numClasses) {

```

Comment:
Code in the comments

Reply:
this should stay in there for the time being and transferred to an appropriate test (new ticket)